fw/cfg.c

```diff
@@ -1771,6 +1809,35 @@ tfw_cfg_set_str(TfwCfgSpec *cs, TfwCfgEntry *e)
 	return 0;
 }
 
+int
+tfw_cfg_set_mem(TfwCfgSpec *cs, TfwCfgEntry *e)
+{
+	long *dest_long;
+	unsigned long val;
+	TfwCfgSpecMem *cse;
+
+	BUG_ON(!cs->dest);
+
+	if (tfw_cfg_check_single_val(e))
```

Comment:
we decide TFW_CFG_CHECK_VAL_N every time

Reply:
In this case `tfw_cfg_check_single_val()` is suitable, because does few checks.